### DISCUSSION TITLE
Finetune Semantic Segmentation with custom model

### DISCUSSION THREAD

**Discussion by sirtris** [PyTorchLightning/lightning-flash#1288] [Q&A] [answered]
Hi,
I have pre-trained my own network backbone using the `ImageEmbedder`. I would now use this model in a down-stream segmentation task. How could I do this? 

For a classification task I would try:
`model = ImageClassifier.load_from_checkpoint("image_classification_model.pt")` 
The problem is that the `SemanticSegmentation()` has no function `load_from_checkpoint`

Thanks

**Comment by ethanwharris** [comment, marked as answer]
Hi @sirtris Thanks for the question! This is a workflow we don't currently support at the moment but is something we would like to support in the future. I'll transfer this discussion to an issue so you can track it :smiley:
